changelogs/fragments/138-postgresql_info_fix_ver_handling.yml

```
@@ -0,0 +1,6 @@
+bugfixes:
+- postgresql_info - Fix extension version handling when it has 0 value (https://github.com/ansible-collections/community.postgresql/issues/137).
+- postgresql_ext - Fix extension version handling when it has 0 value (https://github.com/ansible-collections/community.postgresql/pull/138).
```

Is the linking intended of issue and pull request?
I think #136 is the issue for `postgresql_ext` and #137 for `postgresql_info`

```suggestion
- postgresql_info - Fix extension version handling when it has 0 value (https://github.com/ansible-collections/community.postgresql/issues/137).
- postgresql_ext - Fix extension version handling when it has 0 value (https://github.com/ansible-collections/community.postgresql/issues/136).
```

---

@tcraxs ah, yes, i somehow missed that there's the postgresql_ext related issue as well:) Committed, thanks!